Soccar [UI]/src/nl/soccar/ui/fx/controller/GameViewFXMLController.java

```diff
@@ -6,24 +6,18 @@
 import javafx.fxml.Initializable;
 import javafx.scene.canvas.Canvas;
 import javafx.scene.layout.AnchorPane;
-import nl.soccar.library.Car;
-import nl.soccar.library.Map;
-import nl.soccar.library.Session;
+import nl.soccar.library.*;
+import nl.soccar.library.enumeration.Duration;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.library.enumeration.Duration'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)